free_thresh: 0.35
image: cave.png
negate: 0
occupied_thresh: 0.65
origin: [-8.0, -8.0, 0.0]
resolution: 0.032
